test/cpp/jit/CMakeLists.txt

```diff
@@ -74,7 +75,7 @@ if(USE_SYSTEM_ONNX)
   target_link_libraries(test_jit PRIVATE onnx_proto onnx)
 endif()
 
-set(JIT_TEST_DEPENDENCIES torch gtest)
```

Comment:
What's the reason for this change? Is it because `jitbackend_test` includes `torch`?

Reply:
Yes. I'm trying to resolve CI failures and testing if this change could help. It turns out not. I may change it back after verification.